docs/manual/creating-a-checker.tex

```diff
@@ -2269,13 +2269,19 @@
 help you to create your checker.
 
 \emph{Type-checking}:
-Do not use \code{instanceof} to determine the class of a \<TypeMirror> or \<Element>,
+Do not use \code{instanceof} to determine the kind of a \<TypeMirror> or \<Element>,
```

Comment:
`instanceof` tests classes, not kinds, so I think this change is not desirable.

Reply:
I think using `class` is (also) confusing because `instanceof` does correctly determine a class or interface.  I rewrote this sentence to be clearer.